src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -73,7 +74,14 @@
             "/queryAggregatedObject", "/queryMissedNotifications", "/query",
             "/rules/rule-check/testRulePageEnabled" }, method = RequestMethod.GET)
     public ResponseEntity<String> getRequests(Model model, HttpServletRequest incomingRequest) {
-        String eiRequestUrl = eiRequestsControllerUtils.getEIRequestURL(incomingRequest);
+        String eiRequestUrl;
```

Comment:
From what I can see there are 4 identical code blocks added here. Maybe it's possible to put them in a method.

Reply:
Thats true,, but its , maybe, only Log message that can be moved out to help method.